packages/plugin-data/src/api/data/tree/exportApi.ts

```diff
@@ -115,8 +115,10 @@ export class ExportApi {
       }
     }
 
-    config.query = config.query.toLowerCase().trim();
-    if (!config.query.startsWith('select')) {
+    config.query = config.query.trim();
+    let startsWithSelectPattern = new RegExp('^select', 'i');
+
+    if (!config.query.match(startsWithSelectPattern)) {
```

Comment:
This could use the `/regex/.test('foo')` form of regex to accomplish the same and eliminates the need for `startsWithSelectPattern` variable.

`if (!/^select/i.test(config.query)) ...`

Reply:
Changes made.